### ISSUE TITLE
VRAM for full-finetuning

### ISSUE THREAD

**Issue by alfredplpl** [fal-ai/f-lite#10]
Great work!

I would like to full-finetuning F-Lite.
But, I cannot full-finetuning it by A6000 (48GB).

How much VRAM do we full-finetuning?

I have tested these options:
- precomputed embeddings (512x512, batch size: 1)
- 8bit adam
- gradient checkpointing
- mixed precision: bf16

Thanks in advance!

**Comment by erwold** [comment]
Not sure if this is the reason, but could you try setting gradient_checkpoint to true in your model’s config.json file?

![Image](https://github.com/user-attachments/assets/2ff5e93a-2554-4d5b-a20c-d00578b592ab)

**Comment by alfredplpl** [comment]
Thank you for your advice.

I modify the json.
But, the training code occured the OOM error.

**Comment by alfredplpl** [comment]
I see.

The 10B model is too big to finetuning.
I will use RTX PRO 6000 (96GB).

Thanks.
